src/components/tooltip/tooltip.js

```diff
@@ -10,6 +36,116 @@ import {h, Component} from 'preact';
  * @extends {Component}
  */
 class Tooltip extends Component {
+  state: Object;
+  _hoverTimeout: ?number;
+  textElement: HTMLSpanElement;
+  nonValidTypes: string[] = [];
+
+  /**
+   * default component props
+   * @type {Object}
+   * @memberof Tooltip
+   */
+  static defaultProps = {
+    type: ToolTipType.Top,
+    isMultiline: true,
+    maxWidth: '240px'
```

Comment:
create const

Reply:
this is kind of constant the way it is now in the defaultProps. it is single place to update the defaults. i think it is clear the way it is now